j_lift/org.lift.io/src/main/java/org/lift/io/FeatureSetBuilder.java

```diff
@@ -0,0 +1,16 @@
+package org.lift.io;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import org.apache.uima.jcas.JCas;
+import org.lift.api.Feature;
+import org.lift.api.LiftFeatureExtrationException;
+
+public class FeatureSetBuilder {
```

Comment:
Wo wird der benutzt bzw wofür wird er gebraucht?

Reply:
Es wird zum Aufrufen von extract()-Funktionen in den Feature-Klassen verwendet. Ich habe es beim Exportieren von Feature-Werten in eine CSV-Datei verwendet. Seine Verwendung hier ist unnötig, daher werde ich ihn entfernen.